src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.container {
```

Comment:
since it's a separate BEM-block it should be in a separate file (even if it's small block)